docs/standard/commandline/get-started-tutorial.md

```diff
@@ -39,17 +39,17 @@ Or
 
 ## Create the app
 
-Create a .NET 6 console app project named "scl".
+Create a .NET 9 console app project named "scl".
 
 1. Create a folder named *scl* for the project, and then open a command prompt in the new folder.
 
 1. Run the following command:
 
    ```dotnetcli
-   dotnet new console --framework net6.0
+   dotnet new console --framework net9.0
```

Comment:
Is the version important? Can we just create a new .NET console app and things will work regardless? The previous release supported standard 2.0, is that still the case with beta 5? Does the code shown in here use some .NET 9-specific feature?